cmd/shield/main.go

```diff
@@ -160,7 +167,7 @@ func main() {
 	*/
 
 	c.HelpBreak()
-	c.Dispatch("list targets", "List available backup targets",
+	c.Dispatch("list targets", "List available backup targets\r\x1b[1A\x1b[1mTARGETS:\n\x1b[0m",
```

Comment:
This breaks the semantics of how help works.  Instead of abusing the implementation of help, we should add a new method to `Command` to do something like this:

```
c.HelpGroup("TARGETS")
c.Dispatch("list targets", ... etc ...)
```